src/OcelotBuilderExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Ocelot.DependencyInjection;
+using Nacos.AspNetCore.V2;
+
+namespace Ocelot.Discovery.Nacos;
+
+public static class OcelotBuilderExtensions
+{
+    public static IOcelotBuilder AddNacos(this IOcelotBuilder builder, string section = "nacos")
+    {
+        builder.Services
+            .AddNacosAspNet(builder.Configuration,section)
+            .AddSingleton(NacosProviderFactory.Get)
+            .AddSingleton(NacosMiddlewareConfigurationProvider.Get);
```

Comment:
*Just a word of caution!*
It might be premature to define this since you were inspired by the Consul provider and [ConsulMiddlewareConfigurationProvider](https://github.com/ThreeMammals/Ocelot/blob/develop/src/Ocelot.Provider.Consul/ConsulMiddlewareConfigurationProvider.cs) implementation. However, please note that this [Consul config provider](https://github.com/ThreeMammals/Ocelot/blob/develop/src/Ocelot.Provider.Consul/ConsulMiddlewareConfigurationProvider.cs) was developed specifically for the `AddConfigStoredInConsul()` feature, correct? 
Therefore, let's keep it in the code for now, the next phase of acceptance testing will reveal the subsequent steps.